src/utils/git.js

```diff
@@ -44,15 +159,13 @@ export function createGitIgnore(projectName) {
  * @param username
  * @param password
  */
-export async function setupGitHub(projectName, projectDescription, username, password) {
-  winston.log('verbose', 'setupGitHub', { username });
+export async function setupGitHub(projectName, projectDescription, credentials) {
+  winston.log('verbose', 'setupGitHub', credentials.username);
 
   try {
-    const githubResponse = await requestGitHubToken(username, password);
     await createGitIgnore(projectName);
-    await createGitHubRepository(projectName, projectDescription, username, password);
-    await initAddCommitAndPush(username, projectName);
-    await deleteGitHubToken(githubResponse.url, username, password);
+    await createGitHubRepository(projectName, projectDescription, credentials.token);
+    await initAddCommitAndPush(credentials.username, projectName, credentials.isTwoFactorAuth);
   } catch (err) {
     winston.log('error', 'setupGitHub failed for some reason', err);
```

Comment:
Instead of leaving it for some reason, we might should have a different error for each method to better determine where the error occurs.

Reply:
I agree, but different issue.